File: code_sandbox/app_test/assets/mock_entry_with_utils.js
const { addNumbers, buildMessage } = require("./utils");

function generate() {
  const a = 4;
  const b = 5;
  const total = addNumbers(a, b);

  console.log("mock js with utils");

  return {
    source: "mock_entry_with_utils.js",
    values: { a: a, b: b },
    total: total,
    message: buildMessage(a, b, total),
  };
}

module.exports = { generate };

File: code_sandbox/app_test/assets/mock_entry_with_utils.py
from typing import Dict, Any
from utils import add_numbers, build_message, summary


def generate() -> Dict[str, Any]:
    a = 4
    b = 5
    total = add_numbers(a, b)

    print("mock py with utils")

    return {
        "source": "mock_entry_with_utils.py",
        "values": summary(a, b, total),
        "total": total,
        "message": build_message(a, b, total),
    }
